src/sqlparser/src/ast/statement.rs

```diff
@@ -249,6 +249,18 @@ impl Parser {
             } else {
                 ConnectorSchema::native().into()
             })
+        } else if connector.contains("iceberg") {
+            let expected = ConnectorSchema::native();
```

Comment:
I am not sure what the `format` and `encode` should be here, but it should not be the native. `native` means the format and encode is the same with Rsingwave's internal representation.
the `FORMAT` means how to get the operation(Insert/Delete) from the file, the `ENCODE` is means how to get the rows containment from the bytes, such as parquet/csv/json.

Reply:
I think actually the issue is that "batch-only source" does not need to resolve the operation from the file... so the streaming connector's “FORMAT” is meaningless for it... And I am not sure about  the encode, does it stored in iceberg's catalog?

Reply:
I also don't think we need encode or format for iceberg.

Reply:
Yes, for iceberg. We don't need any format or encoding, because they are self-explanatory and users don't need to specify any format and encoding. We could introduce another Format works like `Native` to meet this requirement.

Reply:
Too many places expect source format and encoding, so I don't want to make them accept an `Option<SourceSchema>`. Introduce a trivial format and encoding for iceberg LGTM.

Reply:
> I think actually the issue is that "batch-only source" does not need to resolve the operation from the file... so the streaming connector's “FORMAT” is meaningless for it... And I am not sure about the encode, does it stored in iceberg's catalog? 

here we just want to load from the iceberg, all chunks are written as `insert` I think. `format plain` should be ok, which means append only. `encode` can be a random placeholder, eg. `iceberg`, to mark it as a spec encode and not for common cases.

Reply:
I am not sure if it can be considered as a kind of `format plain`... especially considering there could be deletion files in Iceberge. I am thinking if we need `ConnectorSchema` for the iceberg connector.